### DISCUSSION TITLE
More meaningful event responses

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#2024] [Anything Else]
> I planned to add a view counter to each content on our site by listening to the response, when the event response is received we just increment the counter on our side. But we are concerned about the difference in data since Plausible will just return `ok` or 202 when an event is received even for bots and spammers, and our system won't know whether the event is actually being recorded or not in Plausible. This will make the stats on our database and Plausible have a number difference.
> 
> I know I can query aggregate with the content URL as a filter but considering the number of the content we have and the request limit of Plausible stats, it's not ideal for us to periodically sync the stats for each individual content we have.
> 
> I hope that Plausible can expose more information in the Event response, at least an indicator of whether the event is actually being recorded or not.

(feedback via email)

**Comment by ukutaht** [comment]
This is by design, because revealing whether we consider something a bot would make it a lot easier to write and verify bots that get past our detection.

I don't know if it's a real issue but I thought better safe than sorry
